.gitlab-ci.yml

```diff
@@ -11,8 +11,9 @@ variables:
   OMPI_MCA_btl_vader_single_copy_mechanism: none
 
 before_script:
-  #- apt-get install -yq ...
   - echo 'localhost slots=8' > /etc/openmpi/openmpi-default-hostfile 
+  - apt-get install -yq intel-mkl-core-c-2020.4-304
+  - ln -s /opt/intel/compilers_and_libraries_2020.4.304/linux/mkl/ /opt/intel/
```

Comment:
better to load `/opt/intel/compilers_and_libraries_2020.4.304/linux/mkl/bin/mklvars.sh` to test discovery from `MKLROOT`-defined location?

Reply:
i had that at first, but this seems more common case